libs/shared/ui/nuverial/src/lib/services/title/title.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import { Title } from '@angular/platform-browser';
+import { BehaviorSubject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class TitleService {
+  private readonly _portalTitle: BehaviorSubject<string> = new BehaviorSubject<string>('');
+  private readonly _htmlTitle: BehaviorSubject<string> = new BehaviorSubject<string>('');
+  public portalTitle$ = this._portalTitle.asObservable();
+  public htmlTitle$ = this._htmlTitle.asObservable();
+
+  constructor(private readonly _ngTitleService: Title) {}
+
+  public setPortalTitle(title: string): void {
+    this._portalTitle.next(title);
+  }
```

Comment:
Rather than having a public method for this what if we make this self contained and handle this in the constructor

```suggestion
  constructor(private readonly _ngTitleService: Title) {
    this._portalTitle.next(this._ngTitleService.getTitle())
  }
```